### PR TITLE
enable graph indices into own directories

### PR DESCRIPTION
**Pull request by frenkye**
I had problem with showing 1 graph with all indices inside, so this will separete indices into own graphs.

Defaults behavior is the same and can be changed through parameter.

### CODE REVIEW

**Comment by asuresh4** [comment]
Hi @frenkye , thank you for the PR. We will have a look and get back.

File: elasticsearch_collectd.py [hunk 1]
```diff
@@ -668,6 +670,7 @@ def configure_callback(conf):
     log.info('host: %s' % c.es_host)
     log.info('port: %s' % c.es_port)
     log.info('es_index: %s' % c.es_index)
+    log.info('es_index_separate_graphx: %s' % c.es_index_separate_graph)
```
Comment (neufeind) [inline elasticsearch_collectd.py:673]: graph instead of graphx?

**Review by neufeind — COMMENTED** [review]
Looks like a good idea to have, since graphs otherwise grow quite large

File: elasticsearch_collectd.py (hunk 1 above)
Comment (frenkye) [reply elasticsearch_collectd.py:673]: Yes, this is typo. I'll try on monday to send fix.

**Comment by atoulme** [comment]
Closing for inactivity. Please reopen if these changes are still needed.

**Pull request closed without merging**